docs/articles_en/openvino-workflow/running-inference/inference-devices-and-modes/npu-device.rst

```diff
@@ -13,27 +13,23 @@ Intel® Core™ Ultra generation of CPUs (formerly known as Meteor Lake). It ena
 you to offload certain neural network computation tasks from other devices,
 for more streamlined resource management.
 
-Note that the NPU plugin is currently available only with the Archive distribution of OpenVINO™
-and you need to :doc:`install a proper NPU driver <../../../get-started/configurations/configurations-intel-npu>`
-to use it successfully.
+NPU Plugin is now available through all relevant OpenVINO distribution channels.
 
 | **Supported Platforms:**
 |   Host: Intel® Core™ Ultra (former Meteor Lake)
 |   NPU device: NPU 3720
 |   OS: Ubuntu* 22.04 64-bit (with Linux kernel 6.6+), MS Windows* 11 64-bit (22H2, 23H2)
 
-
-| **Supported Inference Data Types**
-| The NPU plugin supports the following data types as inference precision of internal primitives:
-|    Floating-point data types: F32, F16O
-|    Quantized data types: U8 (quantized models may be int8 or mixed FP16-INT8)
-|    Computation precision for the HW is FP16.
-|
-| For more details on how to get a quantized model, refer to the
-  :doc:`Model Optimization guide <../../model-optimization>` and
-  :doc:`NNCF tool quantization guide <../../model-optimization-guide/quantizing-models-post-training/basic-quantization-flow>`.
+NPU Plugin needs an NPU Driver to be installed on the system for both model compilation and execution.
+Please follow the instructions below to install the latest NPU drivers:
+- Windows driver: https://www.intel.com/content/www/us/en/download/794734/intel-npu-driver-windows.html
+- Linux driver: https://github.com/intel/linux-npu-driver/releases
 
 
+The plugin uses the graph extension API exposed by the driver to convert the OpenVINO specific representation
```

Comment:
This is to provide a short context of compiler being part of the driver since later on we describe UMD Model Caching.